packs/data/feats.db/garudas-squall.json

```diff
@@ -33,7 +33,7 @@
                     "criticalSuccess"
                 ],
                 "selector": "bow-weapon-group-damage",
-                "text": "If the target of the critical hit is adjacent to a surface, it gets stuck to that surface by the missile. The target is @UUID[Compendium.pf2e.conditionitems.Immobilized]{Immobilized} and must spend an Interact action to attempt a @Check[type:athletics|dc:10] check to pull the missile free; it can't move from its space until it succeeds. The creature doesn't become stuck if it is incorporeal, is liquid (like a water elemental or some oozes), or could otherwise escape without effort.",
+                "text": "PF2E.Item.Weapon.CriticalSpecialization.bow",
```

Comment:
This should just be a CritSpec RE for bows

Reply:
Didn't notice it was a feat.  Fixed.